donate/payments/tasks.py

```diff
@@ -246,8 +348,7 @@ def process_charge_succeeded(event):
         elif 'glassroomnyc' in metadata:
             project = 'glassroomnyc'
 
-        send_to_sqs({
-            'data': {
+        donation_data = {
```

Comment:
Since this function seemed to have been formatting its SQS payload slightly different, I changed the function to instead create an object of the SQS payload, and send the data to SQS how it is originally formatted.

Then, if the donation receipt method is set to the donate stack, we send the donation data to the process_receipt function, where we can manipulate the transaction data as needed to fit the Acoustic API